### ISSUE TITLE
[Question] Time_Of_Day with different days

### ISSUE THREAD

**Issue by NoThreat** [SmartlyDressedGames/Unturned-3.x-Community#1769]
I want to make object visible only from 22:00 to 8:00.
86400 - Full day (0:00)
43200 - Noon (12:00)

I can see, that time after 0:00 resets to 0 value. So how to make my idea? Or it's impossible to make time conditions between two different days?

**Conditions 2
Condition_0_Type Time_Of_Day
Condition_0_Second 79200
Condition_0_Logic Greater_Than_Or_Equal_To

Condition_1_Type Time_Of_Day
Condition_1_Second 28800
Condition_1_Logic Less_Than_Or_Equal_To**

**Comment by SDGNelson** [comment]
You are correct that it is not possible at the moment. Unfortunately it is a stupid limitation of the conditions feature for now.

In the future a feature might be added allowing the OR vs AND logic to be specified like: "(0 | 1) & 2"

**Comment by NoThreat** [comment]
Will wait until update then. Thanks.

**Comment by SDGNelson** [comment]
Closing because the issues list was not really intended for feature requests. Sorry to close this discussion - I do WANT to fulfill all (most?) requests, but it is a matter of prioritization. Constantly being reminded of all the pending requests when checking issues is stressful. I will keep the request in mind. On GitHub maybe the Ideas section of Discussions would be the best place.
